### PR TITLE
make certbot argument --renew-by-default optional

### PR DESCRIPTION
**Pull request by gronke**
The boolean role variable `letsencrypt_force_renew` toggles the `--renew-by-default` certbot argument.

### CODE REVIEW

**Comment by jaywink** [comment]
Mind rebasing this on top of the latest merges?

File: defaults/main.yml [hunk 1]
```diff
@@ -5,3 +5,7 @@ letsencrypt_email: yourmail@example.com
 letsencrypt_domain: example.com
 # Always request www. also?
 letsencrypt_request_www: true
+# Force Certificate Reneval
+letsencrypt_force_renew: false
```
Comment (jaywink) [inline defaults/main.yml]: Can we keep the old default behaviour and have this `true`?

**Comment by gronke** [comment]
done

**Comment by jaywink** [comment]
Thanks!

**Pull request merged**